app/config/services/services.xml

```diff
@@ -213,5 +213,8 @@
 
         <!-- Deputy -->
         <service id="AppBundle\Deputy\DistrictLoader" public="true" />
+
+        <!-- Newsletter -->
+        <service id="AppBundle\Newsletter\NewsletterSubscriptionProcess"/>
```

Comment:
It seems there is already a section for it https://github.com/EnMarche/en-marche.fr/blob/master/app/config/services/community.xml#L6. What about removing those and registering the namespace here?